src/index.html

```diff
@@ -11,7 +11,189 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      method="POST"
+      action="http://localhost:3000/api">
+
+      <fieldset class="inputs-block">
+        <legend>Personal information</legend>
+
+        <lable class="input-row">
+          Surname:
+
+          <input
+          type="text"
+          name="surnname"
+          autocomplete="off"
+          >
+        </lable>
+
+        <lable class="input-row">
+          Name:
+
+          <input
+          type="text"
+          name="name"
+          autocomplete="off"
+          >
+        </lable>
+
+        <lable class="input-row">
+          How old are You?
+
+          <input
+          type="number"
+          name="age"
+          min="1"
+          max="100"
+          value="12"
+          >
+        </lable>
+
+        <lable class="input-row">
+          Full date of birth:
+
+          <input
+          type="date"
+          name="birth"
+          >
+        </lable>
+
+        <lable class="input-row">
+          I accept the term of the agreement
+          <input
+          type="checkbox"
+          name="terms"
+          >
+        </lable>
+
+      </fieldset>
+
+
+      <fieldset class="inputs-block">
+        <legend>Registration</legend>
+
+        <lable class="input-row">
+          E-mail:
+          <input
+          type="email"
+          name="email"
+          placeholder="email@example.com"
+          >
+        </lable>
+
+        <lable class="input-row">
+          Password:
+          <input
+          type="password"
+          name="password"
+          required
+          minlength="1"
+          maxlength="30"
+          >
+        </lable>
+      </fieldset>
+
+      <fieldset class="inputs-block">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="input-row">
+          Do you love cats?
+
+          <label>
+            <input
+            type="radio"
+            name="cats"
+            value="yes"
+            >
+            Yes
+          </label>
+          <label>
+            <input type="radio" name="name"
```

Comment:
Change the input name attribute from 'name' to 'cats' to match the other radio button in the group.